Source/WebCore/platform/ContextMenuItem.h

```diff
@@ -142,6 +142,11 @@ enum ContextMenuAction {
     ContextMenuItemTagMediaPlayPause,
     ContextMenuItemTagMediaMute,
     ContextMenuItemTagDictationAlternative,
+#if ENABLE(PDFJS)
+    ContextMenuItemPDFSinglePageContinuous,
+    ContextMenuItemPDFTwoPages,
+    ContextMenuItemPDFTwoPagesContinuous,
+#endif
```

Comment:
We need to add these values at the end for binary compatibility.

I also wonder if the `ENABLE(PDFJS)` is actually needed around these values. It might be clear to just omit the build flag. Any thoughts @whsieh?

Reply:
I think it would be better to keep the build flag, so it's easier to find/delete PDFJS related code.

Reply:
I guess the solution here would be to include ContextMenuItemLastNonCustomTag inside an if else macro? Inside the if it would point to the PDFJS enum and otherwise it would be as it is in the else branch?

Reply:
Yes, but using a build flag here would also cause binary compatibility issues if we added new values at the end, and then the value of the flag were to change.

Reply:
I removed these build flags and moved the new enum values down. These values will now always be declared, but I still guarded sections of the code that used them.